pkg/jobrunaggregator/jobrunaggregatoranalyzer/junit.go

```diff
@@ -187,14 +190,22 @@ func aggregateTestCase(combined *junit.TestCase, jobName, toAddJobRunID string,
 	switch {
 	case toAdd.FailureOutput != nil:
 		humanURL := jobrunaggregatorapi.GetHumanURL(jobName, toAddJobRunID)
+		testCaseFailure := TestCaseFailure{
+			JobRunID:       toAddJobRunID,
+			HumanURL:       humanURL,
+			GCSArtifactURL: jobrunaggregatorapi.GetGCSArtifactURL(jobName, toAddJobRunID),
+		}
+		if jobrunaggregatorlib.IsDisruptionTest(toAdd.Name) {
+			disruptionSeconds, err := jobrunaggregatorlib.GetOutageSecondsFromMessage(toAdd.FailureOutput.Message)
```

Comment:
this should be updated now that we have separate durations provided.  Possible followup if this is close enough to merge this afternoon.

Reply:
What do you mean by separate durations?